build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -162,6 +162,13 @@ disableTXChecksumOffload: {{ .Values.disableTXChecksumOffload }}
 # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
 defaultMTU: {{ .Values.defaultMTU }}
 
+# packetInRate defines the OVS controller packet rate limits for different
+# features. All features will apply this rate-limit individually on packet-in
+# packets sending to antrea-agent. The number stands for the rate as packets per
+# second(pps) and the burst/queueSize will be automatically set to 2 times of it.
```

Comment:
I think just "burst size" instead of "burst/queueSize"

Reply:
Done.